### PR TITLE
Freesurfer ROI Inputs

### PR DESCRIPTION
**Pull request by swashcap**
Asana task: https://app.asana.com/0/37131652076106/167594259597250

Related:
- MRN-Code/coinstac#21
- MRN-Code/decentralized-laplacian-ridge-regression#6

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -122,6 +122,15 @@ class FreeSurfer {
...
 }
 FreeSurfer.prototype.schema = schema.schema;
+
+/**
+ * Valid fields.
+ * @static
+ *
+ * @const {string[]}
+ */
+FreeSurfer.validFields = schema.validFields;
```
Comment (swashcap) [inline src/index.js:132]: This merely exports the schema (valid FreeSurfer features) so they’re usable in MRN-Code/decentralized-laplacian-ridge-regression as inputs.

**Comment by swashcap** [comment]
**Todo:** We’ll need to bump this module’s version and publish.

**Comment by rssk** [comment]
:+1:

**Comment by swashcap** [comment]
Bumped in fa1d45177b954341780e6edc3ea06d60279ba55d.

**Pull request merged**